content/en/serverless/azure_app_services.md

```diff
@@ -135,6 +135,10 @@ Console.WriteLine(ex);
 DogStatsd.Increment("sample.startup");
 ```
 
+<div class="alert alert-info">Note that if you want to send custom metrics while disabling tracing altogether, you can set the following variables in your application's config:
+  <p class="alert alert-info">- Set the `DD_TRACE_ENABLED` to false.</p>
+  <p class="alert alert-info">- Set `DD_AAS_ENABLE_CUSTOM_METRICS` to true.</p>
```

Comment:
```suggestion
  <p class="alert alert-info">- Set `DD_TRACE_ENABLED` to `false`.</p>
  <p class="alert alert-info">- Set `DD_AAS_ENABLE_CUSTOM_METRICS` to `true`.</p>
```

Reply:
Thanks, this made me realize that I didn't push the last version of my code (div instead of paragraphs).
Fixed in 0d7b9c5ffb638618c789d280c59c44b4aed688c6